SplunkAppForWazuh/appserver/static/js/controllers/agents/overview/agentsOverviewCtrl.js

```diff
@@ -35,6 +35,8 @@ define(['../../module'], function (app) {
     $onInit() {
       if (this.agent.length && typeof this.agent[0] === 'object' && this.agent[0].data && typeof this.agent[0].data.data === 'object') {
         this.scope.agent = this.agent[0].data.data
+        //Check OS type
+        this.scope.isLinux = this.agent[0].data.data.os.uname.includes("Linux") ? true : false
```

Comment:
This operation is unsafe because `.data.data` might be undefined. Also an agent might not have `os` field because it's a never connected.

In addition, `some ? true : false` doesn't make sense, please use just `some` or `!!some` if you want the boolean value. Keep in mind that `.includes(..)` already returns a boolean.